src/domain/accessibilityStatement/AccessibilityStatement.tsx

```diff
@@ -0,0 +1,519 @@
+              </li>
+            </ul>
+            <p>
+              An effort will be made to correct the shortcomings listed in this
+              statement by 30.04.2024.
+            </p>
+
+            <h2>How have we tested accessibility?</h2>
+            <p>
+              The observations in this accessibility statement are based on a
+              third-party assessment of whether the online service meets legal
+              requirements.
+            </p>
+            <p>The online service was published on 28 November 2023.</p>
+            <p>This statement was prepared on 21 November 2023.</p>
+            <p>The statement was last updated on 21 November 2023.</p>
+            <p>
+              <ExternalLink href="https://www.finlex.fi/fi/laki/alkup/2019/20190306">
+                Act on the Provision of Digital Services (306/2019)
+              </ExternalLink>{' '}
+              (in Finnish and Swedish)
+            </p>
+            <p>
+              <ExternalLink href={accessibilityRequirementsWebsiteUrl}>
+                Accessibility Requirements website
+              </ExternalLink>{' '}
+              (in Finnish and Swedish)
+            </p>
+          </>
+        );
+      case 'fi':
+        return (
+          <>
+            <h1>Linked Events verkkopalvelun saavutettavuus</h1>
+            <p>
+              Tässä saavutettavuusselosteessa kerrotaan, miten Linked Events
+              verkkopalvelu noudattaa lakia digitaalisten palvelujen
+              tarjoamisesta, mitä puutteita palvelun saavutettavuudessa on ja
+              miten voit antaa meille palautetta saavutettavuusongelmista.
+            </p>
+            <p>
+              Tästä palvelusta vastaa Helsingin kaupunki. Haluamme, että
+              mahdollisimman moni käyttäjä pystyy käyttämään digitaalisia
+              palveluitamme. Otamme saavutettavuuden huomioon digitaalisten
+              palvelujemme kehityksessä.
+            </p>
+            <h2>Miten saavutettava palvelu on?</h2>
+            <p>
+              Tämä palvelu täyttää osin lain vaatimat A- ja AA-tason
+              saavutettavuuskriteerit (WCAG-kriteeristö 2.1). Palvelussa on
+              joitakin saavutettavuuspuutteita, jotka on kuvattu tarkemmin
+              alempana.
+            </p>
+            <h2>Huomasitko puutteita saavutettavuudessa?</h2>
+            <p>
+              Pyrimme jatkuvasti parantamaan palvelun saavutettavuutta. Jos
+              löydät ongelmia, joita ei ole kuvattu tällä sivulla, ilmoita
+              niistä meille ja teemme parhaamme puutteiden korjaamiseksi.
+              Vastaamme 14 päivän kuluessa.
+            </p>
+            <p>
+              Voit ottaa yhteyttä{' '}
+              <Link to={`/${locale}${ROUTES.SUPPORT_CONTACT}`}>
+                verkkolomakkeella
+              </Link>{' '}
+              tai sähköpostitse osoitteella{' '}
+              <a href={`mailto:${SUPPORT_EMAIL}`}>${SUPPORT_EMAIL}</a>
```

Comment:
Extra $ sign:
<img width="257" alt="Screenshot 2023-12-01 at 11 12 00" src="https://github.com/City-of-Helsinki/linkedcomponents-ui/assets/66477579/2410090e-6330-4eb6-bbbc-d56423ec33d5">

Reply:
Good catch. Removed the extra $.